README.md

```diff
@@ -0,0 +1,238 @@
+- [ ] Five+ days forecast
+- [ ] Write some tests (of course)
+
+<h3 id="built-with">Built with</h3>
+
+- Core
+  - [React Native](https://reactnative.dev/) - A framework for building native apps with React
+- Navigation
+  - [React Navigation](https://reactnavigation.org/) - Routing and navigation for your React Native apps
+- Debugging
+  - [Reactotron](https://github.com/infinitered/reactotron) - Reactotron is a macOS, Windows, and Linux app for inspecting your React JS and React Native apps
+- Styling
+  - [Styled Components](https://styled-components.com/) - Use the best bits of ES6 and CSS to style your apps without stress
+- HTTP Comunication
+  - [Axios](https://github.com/axios/axios) - Promise based HTTP client for the browser and node.js
+- Type Checking
+  - [prop-types](https://github.com/facebook/prop-types) - Runtime type checking for React props and similar objects
+- Linting
+  - [ESLint](https://github.com/eslint/eslint) - Find and fix problems in your JavaScript code
+  - [Prettier](https://prettier.io/) - Prettier is an opinionated code formatter
+- Extra
+  - [react-native-gesture-handler](https://github.com/software-mansion/react-native-gesture-handler) - Declarative API exposing platform native touch and gesture system to React Native
+  - [babel-plugin-root-import](https://github.com/entwicklerstube/babel-plugin-root-import) - Babel plugin to add the opportunity to use import and require with root based paths
+  - [eslint-config-airbnb](https://github.com/airbnb/javascript) - A mostly reasonable approach to JavaScript
+
+<br />
+
+<h2 id="contributing">🤔 Contributing</h2>
+
+> To get started...
+
+### Step 1
+
+- 🍴 Fork this repo!
+
+### Step 2
+
+- 👯 Clone this repo to your local machine using `git clone https://github.com/lucasmontano/openweathermap-reactnative.git`
+
+### Step 3
+
+- 🎋 Create your feature branch using `git checkout -b my-feature`
+
+### Step 4
+
+- ✅ Commit your changes using `git commit -m 'feat: My new feature'`;
+
+### Step 5
+
+- 📌 Push to the branch using `git push origin my-feature`;
+
+### Step 6
+
+- 🔃 Create a new pull request
+
+After your Pull Request is merged, can you delete your feature branch.
+
+<br />
+
+<h2 id="support">📌 Support</h2>
+
+Reach out to me at one of the following places!
+
+- Twitter at [@lucas_montano](https://twitter.com/lucas_montano)
+- Instagram at [@lucasmontano](https://www.instagram.com/lucasmontano/)
+- Linkedin at [Lucas Montano](https://www.linkedin.com/in/lucasmontano/)
+- Youtube at [Lucas Montano](https://www.youtube.com/lucasmontano)
+
+<br />
+
+<h2 id="license">📝 License</h2>
+
+This project is licensed under the MIT License - see the [`LICENSE`](LICENSE) file for details.
+
+<br />
+
+<h2 id="other-platforms">⚒ Other Platforms</h2>
+
+Open Weather on Other Platforms:
+
+- Android: https://github.com/lucasmontano/openweathermap
+- iOS: https://github.com/lucasmontano/openweathermap-ios
+- Flutter: https://github.com/lucasmontano/openweathermap-flutter
+
```

Comment:
The React Native repository without expo is missing here.

Reply:
It's done, man. The link of repository have been added to code. Thanks for notify me about this.